composer.json

```diff
@@ -39,8 +39,5 @@
         "psr-4": {
             "Amp\\Cache\\Test\\": "test"
         }
-    },
-    "conflict": {
-        "amphp/file": "<0.2 || >=2"
```

Comment:
Actually we still need the conflict. It should conflict with amphp/file <2 and possibly also >=3

Reply:
Will gladly add. 

I was going to ask why the require constraints would not take care of the lower version, but I am guessing this is because it's a dev requirement?

Reply:
yeah exactly... it's an optional dependency that's why we're adding it to conflict instead of require